docs/datasheets/dbc-abstracts.md

```diff
@@ -0,0 +1,153 @@
+---
+pretty_name: dbc-abstracts
+language:
+  - da
+  - en
+  - "no"
+  - sv
+  - ty
```

Comment:
ty?

Reply:
Oops. I de-danished the country code, relying on ISO-639-1 instead.